node/fxutil/options.go

```diff
@@ -131,6 +131,11 @@ func OverrideSupply(vals ...interface{}) Option {
 				refVal, tp = refVal.Elem(), tp.Elem()
 			}
 
+			if !refVal.IsValid() {
+				// if there was nil - ignore
+				continue
```

Comment:
Is it okay to silently ignore an invalid thing here?

Reply:
Good question. All the __really__ invalid cases are checked and errors for them are thrown above the line. Regarding this case, it is basically `nil` and it is usually fine to allow it to be passed in the optional pattern, e.g  `WithSomething(nil)`, so you don't need to check for `if something != nil { WithSomething(something) }`.

Reply:
Ah okay. Can you document this in a comment?